Client/Frontend/Browser/TabManager.swift

```diff
@@ -327,6 +327,13 @@ class TabManager : NSObject {
         return -1
     }
 
+    func getTab(webView: WKWebView) -> Browser? {
```

Comment:
Again, I'm not 100% satisfied with this new method but couldn't find a better way. The `WKNavigationDelegate` methods only include the web view but not the tab. We do, however, need the tab in order to store the last user agent override for the action sheet toggling. This cannot be stored on the web view itself without subclassing it.